src/Features/Core/Portable/Completion/CompletionService_GetCompletions.cs

```diff
@@ -243,14 +243,11 @@ private static bool HasAnyItems(CompletionContext cc)
 
         private CompletionList MergeAndPruneCompletionLists(
             ImmutableArray<CompletionContext> completionContexts,
-            TextSpan defaultSpan,
             in CompletionOptions options,
             bool isExclusive)
         {
-            // See if any contexts changed the completion list span.  If so, the first context that
-            // changed it 'wins' and picks the span that will be used for all items in the completion
-            // list.  If no contexts changed it, then just use the default span provided by the service.
-            var finalCompletionListSpan = completionContexts.FirstOrDefault(c => c.CompletionListSpan != defaultSpan)?.CompletionListSpan ?? defaultSpan;
```

Comment:
This is unnecessary, we created those completionContexts with defaultItemSpan [here](https://github.com/dotnet/roslyn/pull/67685/files#diff-dfe33545f5dc5eeb46be6040c59045396311032cd9cc0f5af0f81e47605de51dL324)